targets/zephyr/src/main-zephyr.c

```diff
@@ -186,11 +150,14 @@ void main (void)
     printf ("Error: could not look up print function, expression results won't be printed\n");
   }
 
-  shell_register_app_cmd_handler (shell_cmd_handler);
-  shell_init ("js> ", commands);
-  /* Don't call jerry_cleanup() here, as shell_init() returns after setting
-     up background task to process shell input, and that task calls
-     shell_cmd_handler(), etc. as callbacks. This processing happens in
-     the infinite loop, so JerryScript doesn't need to be de-initialized. */
-} /* main */
+  while (1)
+  {
+    char *s;
+    printf("js> ");
+    fflush(stdout);
+    s = zephyr_getline ();
+    shell_cmd_handler (s);
```

Comment:
The test is something i quite use for automatic testing and show that Javascript works without having to type a long function. But i am not particularly attached to any code there :D